azure-devops/app-projects/03_service_connections_aks.tf

```diff
@@ -0,0 +1,38 @@
+resource "azuredevops_serviceendpoint_kubernetes" "aks-dev" {
+  depends_on            = [azuredevops_project.project]
+  project_id            = azuredevops_project.project.id
+  service_endpoint_name = "${var.prefix}-aks-dev"
+  apiserver_url         = module.secrets.values["aks-apiserver-url"].value
+  authorization_type    = "ServiceAccount"
+  service_account {
+    # base64 values
+    token   = module.secrets.values["aks-azure-devops-sa-token-dev"].value
+    ca_cert = module.secrets.values["aks-azure-devops-sa-cacrt-dev"].value
```

Comment:
this one `aks-azure-devops-sa-cacrt-dev` ? 
isn't declared in `secret_prod.tf` ?